File: Day 014/1672. Richest Customer Wealth.java
class Solution {
    public int maximumWealth(int[][] accounts){
        int r = accounts.length;
        int c = accounts[0].length;
        int ms =0;
        for (int i =0;i < r;i++){
            int s=0;
            for (int j =0;j<c;j++){
                s+=accounts[i][j];
            }
            ms = Math.max (s, ms);
        }
        return ms;
    }
}

File: Day 014/1672. Richest Customer Wealth.cs
public class Solution {
    public int MaximumWealth(int[][] accounts) {
        int ms = 0;
        for (int i = 0; i < accounts.Length; i++) {
            int s = 0;
            for (int j = 0; j < accounts[i].Length; j++) {
                s += accounts[i][j];
            }
            if (s > ms) {
                ms = s;
            }
        }
        return ms;
    }
}